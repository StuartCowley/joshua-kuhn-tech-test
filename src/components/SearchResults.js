import React from "react";

import "../styles/search-results.css";

const SearchResults = ({ results }) => {
  if (!results.length) {
    return <p>No results found!</p>;
  } else {
    return (
      <div className="image-container">
        {results.map((image, index) => (
          <img
            key={index}
            className="space-image"
            src={image}
            alt="spaceImage"
          />
        ))}
      </div>
    );
  }
};

export default SearchResults;
