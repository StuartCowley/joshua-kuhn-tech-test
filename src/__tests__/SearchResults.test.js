import React from "react";
import { render } from "@testing-library/react";
import SearchResults from "../components/Search";

describe("SearchResults", () => {
  const { asFragment } = render(<SearchResults />);

  it("renders correctly", () => {
    expect(asFragment()).toMatchSnapshot();
  });
});
