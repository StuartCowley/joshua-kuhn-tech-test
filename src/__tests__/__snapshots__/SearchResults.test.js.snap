// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchResults renders correctly 1`] = `
<DocumentFragment>
  <form
    class="search-form"
  >
    <input
      class="search-input"
      type="text"
    />
    <button
      class="search-button"
      type="submit"
    >
      Search
    </button>
  </form>
</DocumentFragment>
`;
